packages/client/src/components/QuestionResult/QuestionResult.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { Text } from '@codement/ui/components/Text/Text';
+import { QuestionResultDiv, IconBorder, StyledIcon, StyledTextDiv } from './QuestionResult.styles';
+
+export interface QuestionResultProps extends React.HTMLAttributes<HTMLDivElement> {
+    state: 'success' | 'error';
+}
+
+export const QuestionResult: React.FC<QuestionResultProps> = ({ state, children }) => {
+  const icon = (state === 'success') ? <StyledIcon icon="check" size="lg" color="white" />
+    : <StyledIcon icon="x" size="lg" color="white" />;
```

Comment:
Is the only difference here the icon? If so, let's move this down into the component, with the ternary operator you've got here in the icon. EG:

```ts
<IconBorder>
   <StyledIcon icon={state === 'success' ? 'check' : 'x'} size="lg" color="white" />
</IconBorder>
```

Reply:
Yes, the only change is the icon so will change the code to how you have shown it. Thank you for the feedback.